boards/arm/stm32h735g_disco/stm32h735g_disco_defconfig

```diff
@@ -26,3 +26,9 @@ CONFIG_PINCTRL=y
 
 # select power supply
 CONFIG_POWER_SUPPLY_DIRECT_SMPS=y
+
+# use LPTIM as clock instead of SYSTICK
+CONFIG_CORTEX_M_SYSTICK=n
```

Comment:
Enabling lptim1 in device tree + CONFIG_PM=y should be sufficient to perform this setting.
Please check how this is achived on other series such as L4.
I don't think there is much to do to get it working similarly on on H7 .

Reply:
done